code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -317,12 +408,10 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 			adjustBruteLoss(30)
 
 /mob/living/simple_animal/hostile/guardian/gib()
-	if(summoner)
-		to_chat(summoner, "[span_danger("<B>Your [src] was blown up!")]</B>")
-		summoner.investigate_log("has been gibbed by an explosion.", INVESTIGATE_DEATHS)
-		summoner.gib()
-	ghostize()
-	qdel(src)
```

Comment:
is this gone? or moved to like, an ex_act

Reply:
this goes against how they are designed, killing them should dust you. you are still revivable from being gibbed. other than that its unnecessary